WKWebViewEditMenuTest/ViewController.m

```diff
@@ -33,6 +33,14 @@ - (void)viewDidLoad {
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(menuDidHide) name:UIMenuControllerDidHideMenuNotification object:nil];
 }
 
+- (BOOL)canResignFirstResponder {
+    return NO;
+}
+
+- (BOOL)canBecomeFirstResponder {
+    return YES;
```

Comment:
These apparently make the `menuWillShow:` etc methods obsolete correct? If so shouldn't those be deleted?